test/test_integration_variable.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI Variable Integration tests
+This test file handles the main test flows for the variable command
+"""
+import io
+import json
+import os
+import tempfile
+import uuid
+from contextlib import redirect_stderr
+from unittest.mock import patch
+
+import Utils
+from conjur.constants import DEFAULT_NETRC_FILE
+from .test_integration_cli import CliIntegrationTest
+from .util.cli_helpers import integration_test
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+from Utils import py_utils as utils
+
+# Not coverage tested since integration tests don't run in
+# the same build step
+class CliIntegrationVariableTest(IntegrationTestCaseBase):  # pragma: no cover
+    DEFINED_VARIABLE_ID = 'one/password'
+    capture_stream = io.StringIO()
+    def __init__(self, testname, client_params=None, environment_params=None):
+        super(CliIntegrationVariableTest, self).__init__(testname, client_params, environment_params)
+
+    # *************** HELPERS ***************
+
+    def setup_cli_params(self, env_vars, *params):
+        self.cli_auth_params = ['--debug']
+        self.cli_auth_params += params
+
+        return self.cli_auth_params
+
+    def setUp(self):
+        self.setup_cli_params({})
+        # Need to configure the CLI and login to perform further commands
```

Comment:
What this comment means ? It is a TODO or is explain to what setup_cli does?

Reply:
It is a comment explaining what this line does but I will change to `# Used to configure the CLI and login to run tests`